api/src/team/infrastructure/repositories/certification-center-membership.repository.js

```diff
@@ -287,6 +288,15 @@ const update = async function (certificationCenterMembership) {
   await knex(CERTIFICATION_CENTER_MEMBERSHIP_TABLE_NAME).update(data).where({ id: certificationCenterMembership.id });
 };
 
+const updateLastAccessedAt = async function ({ userId, certificationCenterId }) {
+  const now = new Date();
+  const knexConnection = DomainTransaction.getConnection();
+
+  await knexConnection(CERTIFICATION_CENTER_MEMBERSHIP_TABLE_NAME)
+    .where({ userId, certificationCenterId })
+    .update({ lastAccessedAt: now, updatedAt: now });
+};
```

Comment:
Permettre de spécifier un `lastAccessedAt` arbitraire, en effet cette fonction du repository pourrait être utilisée par un script de backfill ou autre, comme dans #11561 : 

```suggestion
const updateLastAccessedAt = async function ({ lastAccessedAt, userId, certificationCenterId }) {
  const knexConnection = DomainTransaction.getConnection();

  await knexConnection(CERTIFICATION_CENTER_MEMBERSHIP_TABLE_NAME)
    .where({ userId, certificationCenterId })
    .update({ lastAccessedAt, updatedAt: new Date() });
};
```